service/discovery/azure/compute.go

```diff
@@ -299,7 +299,7 @@ func (d *azureComputeDiscovery) discoverFunctionsWebApps() ([]voc.IsCloudResourc
 
 				// Also add function/web app slots
 				var slots []voc.IsCloudResource
-				slots, err = d.discoverSlots(site, config, isWebApp)
+				slots, err = d.discoverSlots(site, isWebApp)
```

Comment:
Shouldn't we discover the slots independently of the previously discovered WebApps/Functions? I think the discoverSlots() should be called outside the if.

Reply:
The `if r != nil` was there before and as far as I understand, we use it because we don't discover all possibilities of `site`s, e.g. `case "app,linux,container"`. So I wanted to make sure to discover slots only if we discover the corresponding base site.